modules/fx/arrow-fx/src/main/kotlin/arrow/fx/IO.kt

```diff
@@ -1018,7 +1018,7 @@ sealed class IO<out A> : IOOf<A> {
     override fun unsafeRunTimedTotal(limit: Duration): Option<A> = unsafeResync(this, limit)
   }
 
-  internal data class Bind<E, out A>(val cont: IO<E>, val g: (E) -> IO<A>) : IO<A>() {
+  data class Bind<E, out A>(val cont: IO<E>, val g: (E) -> IO<A>) : IO<A>() {
```

Comment:
```suggestion
  internal data class Bind<E, out A>(val cont: IO<E>, val g: (E) -> IO<A>) : IO<A>() {
```

Reply:
You can go to the file in the Files Changed tab, click on the `...` and select `Edit File`. AFAIK it should commit to this PR directly :D